BC-MAY-21-ROR/magma-clothing-estore-CarlosSe
# Feature/create ror application

Quick Info
---
Create basic Ruby on Rails application

Migrations? :-1:

What does this change?
----
- Creates a Ruby on Rails application
- Adds Rspec testing suite
- Configures CircleCI 

How do you manually test this?
----
Run on the command line console
> bundle exec rails server

then browse to http://127.0.0.1:3000 you'll see the RoR landing page

Screenshots
----

### Desktop
![Screenshot 2022-08-17 at 15-25-58 Ruby on Rails](https://user-images.githubusercontent.com/44456304/185236406-78c66cae-238c-484c-844e-b499faa144ce.png)
